tests/OTAnalytics/plugin_parser/test_otvision_parser.py

```diff
@@ -627,6 +630,78 @@ def test_serialize_events(
         assert event_list_file.exists()
 
 
+class TestCachedVideo:
+    def test_cache_frames(self, test_data_tmp_dir: Path) -> None:
```

Comment:
Can we also add tests that the `CachedVideo` classes' `get_path` and `to_dict` methods are properly delegated to `other` as well?

Reply:
Fixed :-)